lightning/src/ln/peer_handler.rs

```diff
@@ -236,6 +236,14 @@ impl error::Error for PeerHandleError {
 	}
 }
 
+/// Trait for subscribing to changes of a peer manager's peer set
+pub trait PeerConnectivityUpdater {
```

Comment:
For naming maybe simply `Connectivity`. Alternative: `Listen` similar to listening for blocks though might cause some problems in bindings if the name is reused.